inkstitch.py

```diff
@@ -2,27 +2,101 @@
 #
 # Copyright (c) 2010 Authors
 # Licensed under the GNU GPL version 3.0 or later.  See the file LICENSE for details.
-import cProfile
-import pstats
-import logging
+
 import os
 import sys
+from pathlib import Path
+import configparser
+import lib.debug_utils as debug_utils
+
+SCRIPTDIR = Path(__file__).parent.absolute()
+
+if len(sys.argv) < 2:
+    # no arguments - prevent accidentally running this script
+    print("No arguments given, continue without arguments?")
```

Comment:
There's really never a reason to run Ink/Stitch directly without arguments.  We should exit here and give a user-friendly error in a pop-up window, because users _do_ click `inkstitch.exe` expecting it to do something.

If you don't want to do that in this PR that's fine, writing a whole wxpython app is probably more than you bargained for. :)

Reply:
I just wanted to be visually neutral. I wasn't sure what library to use. In 2D, I prefer Qt, but Inkscape uses GTK, Python defaults to Tk, and Inkstitch is built on wx, which unfortunately does not support gradient lines, which is a peculiar choice for an embroidery program where gradient lines are essential. Then there's the addition of Node.js with Electron and Chromium.
I'll add a dialog in wx.